tools/ci/docker/linux/Dockerfile

```diff
@@ -13,7 +13,7 @@
 # License for the specific language governing permissions and limitations
 # under the License.
 
-FROM ubuntu:20.04 AS builder-base
+FROM ubuntu:22.04 AS builder-base
```

Comment:
@xiaoxiang781216 I tried to build image locally and the first error is fixed by adding `bzip2` after `xz-utils`.
But even with that I can't build an image because of error:
```
#54 446.3 /tools/renesas-tools/source/gcc/gcc/reload1.c: In function 'void init_reload()':
#54 446.3 /tools/renesas-tools/source/gcc/gcc/reload1.c:89:24: error: use of an operand of type 'bool' in 'operator++' is forbidden in C++17
#54 446.3    89 |   (this_target_reload->x_spill_indirect_levels)
#54 446.3       |   ~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~
#54 446.3 /tools/renesas-tools/source/gcc/gcc/reload1.c:443:7: note: in expansion of macro 'spill_indirect_levels'
#54 446.3   443 |       spill_indirect_levels++;
#54 446.3       |       ^~~~~~~~~~~~~~~~~~~~~
#54 446.3 /tools/renesas-tools/source/gcc/gcc/reload1.c: In function 'void elimination_effects(rtx, machine_mode)':
#54 446.3 /tools/renesas-tools/source/gcc/gcc/reload1.c:3055:7: warning: this statement may fall through [-Wimplicit-fallthrough=]
#54 446.3  3055 |       if (code == POST_MODIFY || code == PRE_MODIFY)
#54 446.3       |       ^~
#54 446.3 /tools/renesas-tools/source/gcc/gcc/reload1.c:3059:5: note: here
#54 446.3  3059 |     case STRICT_LOW_PART:
#54 446.3       |     ^~~~
#54 446.4 make[2]: Leaving directory '/tools/renesas-tools/build/gcc/gcc'
#54 446.4 make[2]: *** [Makefile:1059: reload1.o] Error 1
#54 446.4 make[1]: *** [Makefile:3895: all-gcc] Error 2
#54 446.4 make[1]: Leaving directory '/tools/renesas-tools/build/gcc'
#54 446.4 make: *** [Makefile:842: all] Error 2
```
I'm not sure, but maybe we can seek for an upgrade to renesas toolchain to move forward with `ubuntu:22.04` based images

Reply:
Adding bzip2 is still needed.